testdata/book/grpc.yml

```diff
@@ -9,7 +14,7 @@ steps:
         headers:
           authentication: tokenhello
         message:
-          name: alice
+          name: "{{ vars.names[0] }}"
```

Comment:
Before the fix, variables were not being expanded and were showing up in the debug log.